pkg/datarepair/checker/checker.go

```diff
@@ -30,31 +30,42 @@ var (
 
 // Config contains configurable values for checker
 type Config struct {
-	Interval time.Duration `help:"how frequently checker should audit segments" default:"30s"`
+	Interval      time.Duration `help:"how frequently checker should audit segments" releaseDefault:"30s" devDefault:"0h0m10s"`
+	IrrdbInterval time.Duration `help:"how frequently irrepairable checker should audit segments" releaseDefault:"15s" devDefault:"0h0m5s"`
+}
+
+// remoteSegmentInfo remote segment information
+type remoteSegmentInfo struct {
+	remoteSegmentsChecked       int64
+	remoteSegmentsNeedingRepair int64
+	remoteSegmentsLost          int64
+	remoteSegmentInfo           []string
```

Comment:
no need to have prefix `remoteSegments` for fields. Type name `remoteSegmentInfo` has information about context.

Reply:
all the structure got changed to use durability stats by an earlier PR.